x-pack/plugins/security_solution/public/detections/components/host_isolation/translations.ts

```diff
@@ -35,3 +35,7 @@ export const RETURN_TO_ALERT_DETAILS = i18n.translate(
   'xpack.securitySolution.endpoint.hostIsolation.returnToAlertDetails',
   { defaultMessage: 'Return to alert details' }
 );
+
+export const RUN_OSQUERY = i18n.translate('xpack.securitySolution.endpoint.osquery.runOsquery', {
```

Comment:
is `host_isolation/translations` a correct file for that?

Reply:
removed 👍